Makefile

```diff
@@ -105,9 +105,9 @@ test: tooldep
 
 extracheck:
 	echo "-- checking if code is gofmt'ed"
-	if [ -n "$$($(GOFMT) -d $(PKGFILES))" ]; then \
+	@if [ -n "$$($(GOFMT) -d $(PKGFILES))" ]; then \
+		"$$($(GOFMT) -d $(PKGFILES))" \
```

Comment:
Why call the same command twice?

Reply:
On failures, print the diff to the user.

Same as we do for `mender` and `mender-connect` (and we were trying to do here but it was broken).